source/deploy/mattermost-changelog.md

```diff
@@ -11,6 +11,72 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 - **Cloud Releases Prior to v9.2**: See the [Mattermost Legacy Cloud Changelog](https://docs.mattermost.com/deploy/legacy-cloud-changelog.html) for details.
 ```
 
+## Release v9.10 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: 2024-07-16**
+
+```{Important}
+If you upgrade from a release earlier than v9.5, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged GitLab plugin version [v1.9.1](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.9.1).
+ - Pre-packaged Mattermost Copilot plugin version [v0.8.1](https://github.com/mattermost/mattermost-plugin-ai/releases/tag/v0.8.1).
+ - Pre-packaged Calls plugin version [v0.28.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.28.1).
+ - Re-designed the user profile popover and improved its performance.
+ - Added banner to prompt users to give desktop notification permissions when opening the app.
+ - Increased the width of the profile picture setting to match other user settings.
+ - Improved screen reader support for emoji picker.
+ - Improved the accessibility of plugin buttons in the channel header.
+
+#### Administration
+ - Extended ``PluginSiteStatsHandler`` to support more advanced visualization types.
+ - Stopped broadcasting ``channel_deleted/channel_restored`` messages from private channels to non-members.
+
+#### Performance
+ - Added page load time to client performance metrics.
+ - Added a metric to track time it takes for the Threads view to load.
+ - Added support for mobile client metrics.
+ - Increased the range of LCP metrics that can be measured.
+ - Added polling of ``getStatusesByIds`` and ``getProfilesByIds`` network calls. The interval of which is configurable with the ``UsersStatusAndProfileFetchingPollIntervalMilliseconds`` configuration variable.
+ - Added defer loading plugins scripts.
+
+### Bug Fixes
+ - Fixed an issue where the ``RefreshPostStats`` job could fail.
+ - Fixed an issue where attempting to create a team with the URL of an existing team showed the wrong error message.
+ - Fixed an issue where ``visibilitychange`` js browser event had not been added for updating the user's current timezone.
+ - Fixed an issue where the last admin could no longer be demoted.
```

Comment:
The last admin in the system, or the last admin added to the system? If it's the last admin in the system, does it make sense to be able to demote that account?